### PR TITLE
Add quick entry for yarn

### PR DESCRIPTION
**Pull request by dandv**

### CODE REVIEW

File: packages/eslint-config-airbnb/README.md [hunk 1]
```diff
@@ -10,7 +10,15 @@ We export three ESLint configurations for your usage.
 
 ### eslint-config-airbnb
 
-Our default export contains all of our ESLint rules, including ECMAScript 6+ and React. It requires `eslint`, `eslint-plugin-import`, `eslint-plugin-react`, and `eslint-plugin-jsx-a11y`.
+Our default export contains all of our ESLint rules, including ECMAScript 6+ and React. It requires `eslint`, `eslint-plugin-import`, `eslint-plugin-react`, and `eslint-plugin-jsx-a11y`. If you don't need React, see [eslint-config-airbnb-base](https://npmjs.com/eslint-config-airbnb-base).
+
+**yarn**
+
+```sh
+yarn add --dev eslint-config-airbnb-base eslint-plugin-import eslint-plugin-react eslint-plugin-jsx-a11
```
Comment (ljharb) [inline packages/eslint-config-airbnb/README.md]: the yarn command should go after the npm commands.

File: packages/eslint-config-airbnb/README.md [hunk 2]
```diff
@@ -10,7 +10,9 @@ We export three ESLint configurations for your usage.
 
 ### eslint-config-airbnb
 
-Our default export contains all of our ESLint rules, including ECMAScript 6+ and React. It requires `eslint`, `eslint-plugin-import`, `eslint-plugin-react`, and `eslint-plugin-jsx-a11y`.
+Our default export contains all of our ESLint rules, including ECMAScript 6+ and React. It requires `eslint`, `eslint-plugin-import`, `eslint-plugin-react`, and `eslint-plugin-jsx-a11y`. If you don't need React, see [eslint-config-airbnb-base](https://npmjs.com/eslint-config-airbnb-base).
+
+If you use yarn, run `yarn add --dev eslint-config-airbnb-base eslint-plugin-import eslint-plugin-react eslint-plugin-jsx-a11`, and see below for npm instructions.
```
Comment (tomkel) [inline packages/eslint-config-airbnb/README.md]: typo in `a11y`

Comment (tomkel) [inline packages/eslint-config-airbnb/README.md]: @ljharb fyi you merged in a typo

Comment (ljharb) [inline packages/eslint-config-airbnb/README.md]: whoops, thanks

**Pull request merged**